### DISCUSSION TITLE
💡 Add support to video versions

### DISCUSSION THREAD

**Discussion by devjhoan** [DonutWare/Fladder#87] [Ideas]
### Summary

In Jellyfin you can group versions of the same video/movie, for example you have 2 video files, one for 1080p and one for 2160p.
If you group a movie with 2 video files in the Jellyfin UI it will look like this

![Image](https://github.com/user-attachments/assets/8e6ab010-f6cd-4c06-b3f2-859c8e348c37)

In Fladder currently in the search engine screen if this movie is grouped with another one, both are shown instead of just one, and if you use the top search engine only one will be shown!

![Image](https://github.com/user-attachments/assets/12522552-2ef6-4cc5-b945-ccb5e82027fb)
![Image](https://github.com/user-attachments/assets/8bfb97f3-0ef6-4ee2-a200-4b060149c61e)

### Basic Example

The idea would be that in the movie you can change the video as is the change of audio or subtitles.
![Image](https://github.com/user-attachments/assets/df8f7e77-72b7-4d64-94b1-cfac13d033c0)

### Drawbacks

None.

**Comment by PartyDonut** [comment]
Great Idea, I moved it towards discussions for now. 

I have a few questions.

The search/view showing different movies is kind of different. I would either expect it to do one of the two not this. 
What happens when in library view you enable recursive?

Also what does the path layout look for this. I'm assuming you use the jellyfin recomendations, or are there different ways to do this?
Best_Movie_Ever (2019)
    ├── Best_Movie_Ever (2019) - [1080P].mp4
    ├── Best_Movie_Ever (2019) - [720P].mp4
    └── Best_Movie_Ever (2019) - [Directors Cut].mp4

**Reply by devjhoan** [reply]
I just have a movies folder

movies
├── Best_Movie_Ever (2019) - [1080P].mp4
└──  Best_Movie_Ever (2019) - [720P].mp4

Jellyfin does not automatically detect this, you have to group the versions yourself. 
![image](https://github.com/user-attachments/assets/2330441a-23c3-4a83-960c-efbb5663af07)

**Reply by michalss** [reply]
this is true but there is official plugin call MergeVersions and this one does it automatically.. It is very imoportant function to have. Since we are using only direct plays, so it does make perfect sence to have it..

**Comment by Simonh2o** [comment]
Would also love to see this implemented, Infuse shows a dialog box with the available video versions (if there are more than one) when you hit play.
My favorite player so far for Windows OS.

**Comment by michalss** [comment]
Please implement this. That is only reason why we still using the original crapp player :(

**Comment by PartyDonut** [comment]
Created a feature ticket for this here #227.
